.devcontainer/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.231.6/containers/javascript-node/.devcontainer/base.Dockerfile
```

Comment:
Not opposed to leaving this devcontainer stuff in the repo, but if we do, would you mind adding some documentation in the readme about what exactly this is for?

Reply:
Sure, I can do that! Also happy to drop it if it doesn't seem useful (I just run all of my projects in Docker locally)

Reply:
I've never used devcontainers and we haven't really experimented with them, so maybe easier to just drop for now if you aren't using?

Reply:
I personally am using devcontainers to work on Library, but I'm a regular contributor so, yeah that sounds good!